npm/svelte/src/mount.ts

```diff
@@ -56,19 +48,24 @@ const getComponentDisplayName = <T extends SvelteComponent>(Component: SvelteCon
  *
  * @see {@link https://on.cypress.io/mounting-svelte} for more details.
  */
-export function mount<T extends SvelteComponent> (
-  Component: SvelteConstructor<T>,
-  options: MountOptions<T> = {},
-): Cypress.Chainable<MountReturn<T>> {
+export function mount (
+  Component: Component<Record<string, any>, Record<string, any>, any>,
```

Comment:
The component signature for svelte completely changed in Svelte 5, where Svelte components are no longer generic classes . Instead, the typing represents something more simple now, which is just a generic type of `Record<string, any>`